CONTRIBUTING.md

```diff
@@ -343,7 +343,8 @@ You can also feel free to do `make format` if you're getting errors related to `
 
 There are several steps that go into a major release
 
-- The GitHub release is created via this [GitHub workflow](https://github.com/osmosis-labs/osmosis/blob/main/.github/workflows/release.yml). The workflow is manually triggered from the [osmosis-ci repository](https://github.com/osmosis-labs/osmosis-ci). The workflow uses the `make build-reproducible` command to create the `osmosisd` binaries using the default [Makefile](https://github.com/osmosis-labs/osmosis/blob/main/Makefile#L99).
+# TODO - osmosis-ci DNE (unless it's private). Currently referencing the PR that removed the release workflow.
+- The GitHub release is handled by [osmosis-ci](https://github.com/osmosis-labs/osmosis/pull/2428).
```

Comment:
It's private. Could you please bring back the old text but replace:

```
https://github.com/osmosis-labs/osmosis/blob/main/.github/workflows/release.yml
```

to
```
https://github.com/osmosis-labs/osmosis-ci/blob/main/.github/workflows/create-release.yaml
```

Reply:
Done 🚀